frontend/src/AppInbusImport.vue

```diff
@@ -0,0 +1,327 @@
+<script setup lang="ts">
+import { ref, watch, onMounted } from 'vue';
+
+import { csrfToken } from './api.js';
+
+class WeekActivity {
```

Comment:
Since these are only type definitions, it should be `interface`, rather than `class`. We don't need to create any instances of it.